src/Clock.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timer = 0;
```

Comment:
for initial value would be better - null

```suggestion
  timer: NodeJS.Timer | null = null;
```